lib/android/audio.nit

```diff
@@ -145,9 +156,33 @@ private extern class NativeSoundPool in "Java" `{ android.media.SoundPool `}
 	new(max_streams, stream_type, src_quality: Int) in "Java" `{
 		return new SoundPool((int)max_streams, (int)stream_type, (int)src_quality);
 	`}
-	fun load_asset_fd(afd: NativeAssetFileDescriptor, priority: Int): Int in "Java" `{ return self.load(afd, (int)priority); `}
-	fun load_id(context: NativeActivity, resid, priority: Int): Int in "Java" `{ return self.load(context, (int)resid, (int)priority); `}
-	fun load_path(path: JavaString, priority: Int): Int in "Java" `{ return self.load(path, (int)priority); `}
+	fun load_asset_fd(afd: NativeAssetFileDescriptor, priority: Int): Int in "Java" `{
+		try {
+			int id =  self.load(afd, (int)priority);
+			return id;
+		}catch(Exception e){
+			return -1;
```

Comment:
Shouldn't you keep that message to display it from the Nit side?

Maybe the sameway that is used in [native_mongodb](https://github.com/privat/nit/blob/master/lib/mongodb/native_mongodb.nit#L106-L111)?

@xymus what is your opinion on that?